substrate/frame/revive/src/exec.rs

```diff
@@ -1488,13 +1538,16 @@ where
 				}
 			}
```

Comment:
that should be gone after merging master https://github.com/paritytech/polkadot-sdk/pull/10302 ?

Reply:
Yes and I removed the comment in https://github.com/paritytech/polkadot-sdk/pull/10166/commits/c40e273e8b3062a80e262995cb4b4dcc7bb69160#diff-ccc183489bcbd5a91f42d7c462c72f4f5993b23dec8869d113ec324aae66692d